DemoApp/app/screens/AppCenterScreen.js

```diff
@@ -77,7 +77,7 @@ export default class AppCenterScreen extends Component {
     startupMode: StartupModes[0],
     userId: '',
     accountId: '',
-    isTokenSet: false
+    isTokenSet: 'Authentication status unknown'
```

Comment:
Since if it isn't bool value anymore - please rename it

Reply:
Renamed it.